cmd/containerd-stress/main.go

```diff
@@ -264,13 +264,12 @@ func serve(c config) error {
 
 func criTest(c config) error {
 	var (
-		timeout     = 1 * time.Minute
-		wg          sync.WaitGroup
-		ctx         = namespaces.WithNamespace(context.Background(), stressNs)
-		criEndpoint = "unix:///run/containerd/containerd.sock"
+		timeout = 1 * time.Minute
+		wg      sync.WaitGroup
+		ctx     = namespaces.WithNamespace(context.Background(), stressNs)
 	)
 
-	client, err := remote.NewRuntimeService(criEndpoint, timeout)
+	client, err := remote.NewRuntimeService(c.Address, timeout)
```

Comment:
Will this work if we don't provide any addr and rely on the default value of the addr flag? From what I remember of `GetAddressAndDialer` I think it'll bail if it can't find unix prefix/protocol.

Reply:
`GetAddressAndDialer` has a fallback mechanism to fallback to `unix`
https://github.com/containerd/containerd/blob/main/integration/remote/util/util_unix.go#L97

But I am happy to also prefix the `unix://` to be more explicit but this might be simpler.

Reply:
My only concern was that, so if it handles it by default I'm good with this change!